examples/vmi-masquerade.yaml

```diff
@@ -40,6 +40,9 @@ spec:
       userData: |-
         #!/bin/bash
         echo "fedora" |passwd fedora --stdin
+        ip -6 addr add fd2e:f1fe:9490:a8ff::2/120 dev eth0
```

Comment:
why is this necessary?

Reply:
We don't have dhcp server for ipv6. Vm ipv6 addresses should be configured manually (via cloud-init, console, etc.) Please refer to the PR description for more details.